### ISSUE TITLE
Hyperparameters for reproducing the results of the paper?

### ISSUE THREAD

**Issue by bangawayoo** [FedML-AI/FedNLP#27]
Hi, thank you for the work.

I am confused regarding the learning rates used in the experiments. 
The README.md (experiments/distributed/transformer_exps/README.md) have different server lr's (0.1, 5e-5) with that of the paper (Section 4.3). I am trying to reproduce some experiments as a baseline, but I am reaching either a higher or lower performance than the reported accuracy.
 
For the seq2seq task (Gigawords), could you report the server and client learning rate? or even better refer me to the wandb project?

Thanks

**Comment by bangawayoo** [comment]
This was answered by the authors in the Slack channel 
![image](https://user-images.githubusercontent.com/36688078/158983818-89c03b30-026b-4556-b8e5-43c77e8ad658.png)
